scripts/mods/msu_settings_manager.nut

```diff
@@ -56,12 +69,20 @@ this.msu_settings_manager <- {
 				if (setting.getValue() != value)
 				{
 					setting.onChangedCallback(value);
+					//sometimes printing the change is undesired, like when data is read during init or on some data types
+					if(_informChange && setting.PrintChange == true){
+						this.MSU.PersistentDataManager.writeToLog("ModSetting", "MSU",  format("%s;%s", settingID, value.tostring()))
+					}
 				}
-				setting.set(value);
+				setting.set(value);	
```

Comment:
make sure you have the editorconfig package on sublime because our editorconfig package is written to make sure this kind of thing doesn't happen.